subprojects/core/src/integTest/groovy/org/gradle/api/tasks/bundling/ArchiveIntegrationTest.groovy

```diff
@@ -687,7 +687,7 @@ class ArchiveIntegrationTest extends AbstractIntegrationSpec {
         "test.tar"  | "tarTree" | "createTar"
     }
 
-    def 'ensure duplicates not included in tar by default'() {
+    def 'emit deprecation warning when duplicates are included in tar for default duplicates strategy'() {
```

Comment:
So `ArchiveIntegrationTest` is about TAR, and `ZipIntegrationTest` is about ZIP? Looks like something we should consolidate.

Reply:
`ArchiveIntegrationTest` is for tar and zip. I moved the duplicates tests to `ArchiveIntegrationTest` so we only need to maintain them in one place.